src/Domain/Tool/Phpunit/PhpUnitTask.php

```diff
@@ -323,10 +381,17 @@ protected function runPhpUnit(): void {
         'phpunit',
         '--verbose',
       ];
-      if ($this->shouldGenerateCodeCoverage()) {
-        $command[] = "--coverage-cobertura={$this->cloverCoverage}";
-        $this->processRunner->addEnvVar("XDEBUG_MODE", "coverage");
+
+      if ($this->shouldGenerateCodeCoverageInClover()) {
+        $command[] = "--coverage-clover={$this->cloverCoverage}";
+      }
+
+      if ($this->shouldGenerateCodeCoverageInCobertura()) {
+        $command[] = "--coverage-cobertura={$this->coberturaCoverage}";
       }
+
+      $this->processRunner->addEnvVar("XDEBUG_MODE", "coverage");
```

Comment:
So this environment variable was previously being set only if Clover coverage was enabled. Now it's being set under all circumstances--including if neither Clover _nor_ Cobertura is being generated. Was that a conscious decision?

Reply:
Fixed